internal/contour/handler.go

```diff
@@ -55,9 +59,13 @@ type EventHandler struct {
 	// seq is the sequence counter of the number of times
 	// an event has been received.
 	seq int
+
+	syncTracker *synctrack.SingleFileTracker
```

Comment:
nit: could we some comments here, explaining the usage/purpose?

Reply:
Sure. Added a brief description.

Reply:
might be useful to add a small note that it is basically a counter that is incremented for each resource that is part of the initial state of the cache and then decremented after the event adding it to the contour cache is handled

Reply:
Thanks. I've improved the comment.